### ISSUE TITLE
Spyder has encountered an internal problem!

### ISSUE THREAD

**Issue by Patrick5** [spyder-ide/spyder#15954]
### What steps will reproduce the problem?

Spyder was open, but minimized.  I got this message when I opened the Spyder window.  I have no idea if this will reproduce the problem, probably not because usually it opens just fine.

### Traceback
```python-traceback
  File "C:\Anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 1419, in show_dpi_change_message
    self.show_dpi_change_message)
TypeError: 'method' object is not connected
```

## Versions

* Spyder version: 4.1.3 
* Python version: 3.7.10
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 8.1

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  4.0.0 (OK)
cloudpickle >=0.5.0            :  1.6.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.19.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  6.0.7 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.2 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.8.0 (OK)
pygments >=2.0                 :  2.8.1 (OK)
pylint >=0.25                  :  2.7.2 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  1.0.2 (OK)
qtconsole >=4.6.0              :  5.0.2 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.7 (OK)
sphinx >=0.6.6                 :  3.5.2 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.9.1 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  22.0.3 (OK)

# Optional:
cython >=0.21                  :  0.29.22 (OK)
matplotlib >=2.0.0             :  3.3.4 (OK)
numpy >=1.7                    :  1.20.1 (OK)
pandas >=0.13.1                :  1.2.3 (OK)
scipy >=0.17.0                 :  1.6.0 (OK)
sympy >=0.7.3                  :  1.7.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @Patrick5, thanks for reporting. I think this is already fixed in our latest Spyder 4 version (4.2.5). Please update by closing Spyder, opening the Anaconda Prompt and running there

    conda update anaconda
    conda install spyder=4.2.5

Let us know if it works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
